### DISCUSSION TITLE
Styling issue with libreoffice conversion through gotenberg

### DISCUSSION THREAD

**Discussion by rockyweng** [gotenberg/gotenberg#541] [Q&A]
Hi all,

I have styling issue with the weird dots on table of content for converting word to pdf from LibreOffice through Gotenberg.

Gotenberg version:

<img width="824" alt="gotenberg" src="https://user-images.githubusercontent.com/14155587/200734633-e1387693-daf1-46fb-96b9-74b7dd3bea77.png">

LibreOffice Headless conversion:

![LibreOffice_Headless](https://user-images.githubusercontent.com/14155587/200734934-89010ca0-a759-4496-bbe3-b7a34920d7e4.png)

Issues:
- There is no dot in the original word document however gotenberg conversion is printing dot
- No page number showing on the gotenberg version but it is showing on the direct conversion from libreoffice headless through CLI

Please let me know if I am missing something.

Thank you.

**Comment by gulien** [comment]
Hello @rockyweng,

What’s your LibreOffice version?

**Reply by rockyweng** [reply]
Hi @gulien ,

I have tried both v7.3.5 and v7.4.1 with the LibreOffice and getting the similar styling issue. 

I have tried 3 uno engine options and no luck to resolve the styling issue with the paging.

The second screenshot was generated by the libreoffice CLI command directly instead of generating through gotenberg.

**Comment by rockyweng** [comment]
Hi @gulien ,

Please try this document. I can reproduce the issue with this document and it is showing weird dots and not able to see the page number for the table of content.

Thank you.

[sample-issue-document.docx](https://github.com/gotenberg/gotenberg/files/9999641/sample-issue-document.docx)

**Reply by gulien** [reply]
What's the LibreOffice CLI command you're using?

**Reply by rockyweng** [reply]
Hi @gulien ,

I tried to run the following and got the issue with the layout as shown in the screenshot above.

curl \
--request POST 'http://localhost:3000/forms/libreoffice/convert' \
--form 'files=@"sample-issue-document.docx"' \
--form 'pdfFormat="PDF/A-1a"' \
-o sample-issue-document.pdf

**Comment by luoxzhg** [comment]
I hava a similar problem.

gotenberg version `7.8`
API : `/forms/libreoffice/convert`

[content table item list number error.docx](https://github.com/gotenberg/gotenberg/files/10577666/default.docx)

origin docx file
![图片](https://user-images.githubusercontent.com/25560573/216567674-5d9e5089-c316-474c-b05e-103412c9e7c0.png)

converted pdf
![图片](https://user-images.githubusercontent.com/25560573/216567833-0f03422a-7bde-445c-8fa9-ab55d6ae57bc.png)
